config/default.yaml

```diff
@@ -0,0 +1,6 @@
+defaults:
```

Comment:
Filename is `default` but config name is `defaults`. Is this intentional one? From consistency aspect, it might be better to align name.

Reply:
Config name of root key is fixed to `defautls`, and default Filename is `config.yaml`.
I rename this to `config/config.yaml`. (I'm sorry but I think it shouldn't be unified)